CMakeLists.txt

```diff
@@ -272,7 +272,7 @@ endif(WITH_LEVELDB)
 
 find_package(snappy REQUIRED)
 
-option(WITH_LZ4 "LZ4 compression support" OFF)
+option(WITH_LZ4 "LZ4 compression support" ON)
```

Comment:
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes, in this case, it would be "cmake: ".